crates/bindings-csharp/Codegen.Tests/fixtures/server/Lib.cs

```diff
@@ -13,7 +13,7 @@ public partial struct CustomStruct
 }
 
 [SpacetimeDB.Type]
-public partial struct CustomClass
+public partial class CustomClass
```

Comment:
Huh when did those become structs? I somehow missed it. That was a bug, especially noticeable since those tests were named CustomClass to specifically differentiate them from CustomStructs with the same contents.

Reply:
Ah yeah I was renaming the other one and saw Class in this name but struct used :)

Reply:
Looks like I added like that from the very beginning. Yay for copy-paste 🤦‍♂️